kinto/core/scripts.py

```diff
@@ -88,3 +91,110 @@ def delete_collection(env, bucket_id, collection_id):
     current_transaction.commit()
 
     return 0
+
+
+OLDEST_FIRST = Sort("last_modified", 1)
+BATCH_SIZE = 25
+
+
+def rebuild_quotas(env, dry_run=False):
+    """Administrative command to rebuild quota usage information.
+
+    This command recomputes the amount of space used by all
+    collections and all buckets and updates the quota records in the
+    storage backend to their correct values. This can be useful when
+    cleaning up after a bug like e.g.
+    https://github.com/Kinto/kinto/issues/1226.
+    """
+    registry = env['registry']
+    settings = registry.settings
+    readonly_mode = asbool(settings.get('readonly', False))
+
+    if readonly_mode:
+        # FIXME: there is the possibility of race conditions if under
+        # normal operation while the rebuild operation
+        # happens. Shouldn't we *enforce* readonly mode?
```

Comment:
I just followed the example of the `delete-collection` script, although I feel like these scripts don't need to obey the same rules about readonly mode.